app/components/edit/permission_files_step_component.html.erb

```diff
@@ -2,22 +2,46 @@
                                    title: t('steps.permission_files.title'),
                                    submission:,
                                    edit_focus_id:,
-                                   done_disabled: false) do |component| %>
+                                   done_disabled: done_disabled?) do |component| %>
   <% component.with_help_content do %>
     <%= t('steps.permission_files.help_html') %>
   <% end %>
 
   <% component.with_body_content do %>
-    <%= render Shared::PermissionFilesStepBodyHelpComponent.new %>
-      <% if permission_files.attached? %>
-        <%= render Shared::PermissionFilesStepBodyTableComponent.new(submission:, with_remove: true) %>
+    <%= helpers.with_submission_form(submission) do |form| %>
+      <div class="flex-glow-1">
```

Comment:
Should this be 

```suggestion
      <div class="flex-grow-1">
```

❓

Reply:
(Or does it not need to be there at all, since `flex-glow-1` was apparently fine?)

Reply:
Good catch. Fixed. Not sure if it's a must to be there, but leaving it for consistency.